service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterStatementMapper.java

```diff
@@ -0,0 +1,56 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import lombok.RequiredArgsConstructor;
+import uk.nhs.adaptors.gp2gp.ehr.utils.DateFormatUtil;
+import uk.nhs.adaptors.gp2gp.ehr.utils.TemplateUtils;
+
+import org.hl7.fhir.dstu3.model.Encounter;
+import org.hl7.fhir.dstu3.model.Period;
+import org.hl7.fhir.dstu3.model.ResourceType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.github.mustachejava.Mustache;
+
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Component
+public class EncounterStatementMapper {
+    private static final Mustache ENCOUNTER_STATEMENT_TO_EHR_COMPOSTION_TEMPLATE =
+        TemplateUtils.loadTemplate("ehr_encounter_statement_to_ehr_composition_template.mustache");
+    private static final String COMPLETE_CODE = "COMPLETE";
+    private static final String DEFAULT_TIME_VALUE = "<center nullFlavour=\"UNK\"/>";
+    private static final String EFFECTIVE_LOW_TEMPLATE = "<low value=\"%s\"/>";
+    private static final String EFFECTIVE_HIGH_TEMPLATE = "<high value=\"%s\"/>";
```

Comment:
ObservationStatementMapper also outputs effectiveTime. Is it possible to share this code?